code/form/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>UZH Login</title>
+    <meta name="description" content="Login to your Student´s Profile">
+    <link rel="stylesheet" href="style.css">
+    <link rel='icon' href='favicon-16x16.ico' type='image/x-icon' />
+  </head>
+  <body>
+    <header>
+      <a href="../home/index.html">
+        <img src="./img/UZH_Website_500x300_20180226-1-1024x614.png" alt="Logo">
+      </a>
+      <nav>
+        <a href="https://app.connect.uzh.ch/apps/id/profile.nsf/main" target="blank">
+          <img src="./img/user_icon_flaticon.png" alt="Icon Settings" title="Settings">
+        </a>
+      </nav>
+    </header>
+    <form action="https://httpbin.org/anything" method="post">
+      <h2>Login</h2>
+      <fieldset>
+        <label>
+          <input type="email" name="E-Mail" required value="felix.muster@uzh.ch" />
+        </label>
+        <label>
+          <input type="password" name="Password" required value="Password" id="password" />
+        </label>
+        <label>
+          <input type="submit" name="Submit" value="Sign In" />
+        </label>
+        <div class="line"></div>
+      </fieldset>
+      <div class="password-reset">
+        <br>
+        <a href="#">Forgot Your Password?</a>
+      </div>
+    </form>
+    <footer>

```

Comment:
If you want to, I think it would look great to put the footer attached to the bottom :) Maybe try it out using 
```
footer {
     position: absolute;
     bottom: 0;
}
```